### DISCUSSION TITLE
error after fail updating  then i press ctrl+alt+f4

### DISCUSSION THREAD

**Discussion by qwazky** [elementary/triage#71] [Q&A]
after updated but failed because the network was disconnected, my screen letters on my screen became bigger. tap on my tuchpad doesn't work, lock screen changes, win+left/right buttons don't work, and sound effects change. i am a new linux user. Please help me. the problem is already attached in the picture

![Screenshot from 2021-06-27 15-22-56](https://user-images.githubusercontent.com/75782974/123538014-bd6e0800-d75c-11eb-8802-2eebeea050f3.png)

![Screenshot from 2021-06-27 15-16-59](https://user-images.githubusercontent.com/75782974/123538028-d4acf580-d75c-11eb-8357-d1a9d4bdf9d6.png)

![IMG_20210627_153301](https://user-images.githubusercontent.com/75782974/123538152-9106bb80-d75d-11eb-942a-16ecb70fb91e.jpg)

all theme is default, and the screen resolution 1366 x 768

![Screenshot from 2021-06-27 15-42-56](https://user-images.githubusercontent.com/75782974/123538295-6a955000-d75e-11eb-886e-e6e54ccca930.png)

**Comment by cassidyjames** [comment]
@lortkazuma can you try opening AppCenter and installing all updates listed there? If that doesn't work, can you try opening the Terminal app and running `sudo apt update; sudo apt dist-upgrade` and seeing if that succeeds?

**Comment by danirabbit** [comment]
Closing as Outdated since OP never responded
